src/main/java/com/artipie/conda/CondaRqPath.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/artipie/LICENSE.txt
+ */
+package com.artipie.conda;
+
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
+
+/**
+ * Anaconda client performs requests of the following formats:
+ * <code>/t/ol-4ee312d8-9fe2-44d2-bea9-053325e1ffd5/my-conda/noarch/repodata.json</code>
+ * Where second part is user token, third is repository/user name depending on the layout,
+ * then follows conda repository architecture (for example: noarch, linux-64, win-64 etc).
+ * This {@link Predicate} implementation tests whether path is such conda path.
+ * @since 0.23
+ */
+public final class CondaRqPath implements Predicate<String> {
```

Comment:
If this class has the only implementation, I'd suggest converting it into `enum` for request path matching, e.g. `enum RqPaths implements Predicate<String>` with `RqPaths.CONDA` field. Or, alternatively, it could be a class with primary constructor accepting `Pattern` if we may need configuring this class later.